packages/editor/src/components/entities-saved-states/entity-record-item.js

```diff
@@ -28,6 +28,27 @@ export default function EntityRecordItem( {
 		return parents[ parents.length - 1 ];
 	}, [] );
 
+	// Handle templates that might use default descriptive titles
+	const entityRecordTitle = useSelect(
+		( select ) => {
+			if (
+				! select( 'core/edit-site' ) ||
```

Comment:
The issue with a reference to the parent `edit-site` package is still unresolved.

Reply:
🤦 
Good catch, will update ASAP!